frontend/src/views/Collection/components/DatasetTab/index.tsx

```diff
@@ -29,11 +28,10 @@ const DatasetTab: FC<Props> = ({ collectionID, visibility, datasets }) => {
 
   const [uploadLink] = useCollectionUploadLinks(collectionID, visibility);
   const [uploadedFiles, setUploadedFiles] = useState({} as UploadedFiles);
+  const { data: collection } = useCollection({ id: collectionID, visibility });
```

Comment:
`DatasetTab` needs access to `collection.accessType`, thus using `useCollection` now to get the collection object